scipy/signal/windows.py

```diff
@@ -693,8 +745,8 @@ def hann(M, sym=True):
         return np.ones(M)
     M, needs_trunc = _extend(M, sym)
 
-    n = np.arange(0, M)
-    w = 0.5 - 0.5 * np.cos(2.0 * np.pi * n / (M - 1))
+    a = [0.5, 0.5]
+    w = _cos_win(M, a)
```

Comment:
With all of these it's probably worth saving a few lines with the equally readable `w = _cos_win(M, a=[...])`